test/ruby/test_compile_prism.rb

```diff
@@ -709,7 +709,11 @@ def test_InterpolatedSymbolNode
 
     def test_InterpolatedXStringNode
       assert_prism_eval('`echo #{1}`')
-      assert_prism_eval('`printf #{"100"}`')
+      if /mswin|ucrt/ =~ RUBY_PLATFORM
+        assert_prism_eval('`echo #{"100"}`')
+      else
+        assert_prism_eval('`printf #{"100"}`')
+      end
```

Comment:
This assertion seems nothing different from the immediately above from the point of parser.
```suggestion
```

Reply:
Sorry, misread.
`#{1}` and `#{"100"}`.

I don't think there is a reason to use a different command.
```suggestion
      assert_prism_eval('`echo #{"100"}`')
```